From 382a378507925b935b130241bd73d2d7e402c006 Mon Sep 17 00:00:00 2001
From: "Jeffrey C. Ollie" <jeff@ocjtech.us>
Date: Tue, 20 Sep 2011 09:12:02 -0500
Subject: [PATCH 09/14] Use  consistently in the Fedora init script

---
 contrib/init.d/rc.redhat.asterisk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/contrib/init.d/rc.redhat.asterisk b/contrib/init.d/rc.redhat.asterisk
index 186fc18..7415138 100755
--- a/contrib/init.d/rc.redhat.asterisk
+++ b/contrib/init.d/rc.redhat.asterisk
@@ -82,7 +82,7 @@ RETVAL=0
 start() {
 	# Check if Asterisk is already running.  If it is, then bug out, because
 	# starting safe_asterisk when Asterisk is running is very bad.
-	VERSION=`${AST_SBIN}/asterisk -rx 'core show version'`
+	VERSION=`$DAEMON -rx 'core show version'`
 	if [ "`echo $VERSION | cut -c 1-8`" = "Asterisk" ]; then
 		echo "Asterisk is already running."
 		exit 1
-- 
1.7.6.2

